content/en/agent/iot/_index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: IoT Agent
+kind: documentation
+beta: true
+further_reading:
+  - link: "/getting_started/agent/"
+    tag: "Documentation"
+    text: "Getting Started with the Agent"
+---
+
+## Overview
+
+The Datadog IoT Agent is a version of the Agent optimized for monitoring IoT devices and embedded applications. Customers use the IoT Agent to monitor a wide variety of devices from digital displays to security devices running image detection algorithms.
+
+<div class="alert alert-warning">
+The IoT Agent is currently available for private beta. Request access by completing the <a href="https://docs.google.com/forms/d/e/1FAIpQLSfaoqr9Pgievq7f2WE6wcQMF1YjsOwiXHbmO2FehUlQwmNu0A/viewform?usp=sf_link">form</a>.
+</div>
+
+## Capabilities
+
+The IoT Agent includes the following system checks. Configuration for IoT devices is identical to other types of hosts.
+
+- [System][1] (includes CPU, IO, load, memory, swap, and uptime)
+- [Disk][2]
+- [Network][3]
+- [Systemd][4]
+- [NTP][5]
+
+The IoT Agent also supports:
+
+- Custom metric collection using an embedded [DogStatsD][6] server
+- Log collection using [tailing files][7], [TCP/UDP][8], and [journald][9]
+
+The IoT Agent does not include the Python interpreter and other integrations pre-packaged with the standard Agent. It also doesn’t support tracing for APM, live process monitoring, or network performance monitoring.
+
+## Setup
+
+### Requirements
+
+The IoT Agent is available as DEB and RPM packages for Linux devices running on x64, arm64 (ARMv8), and ARMv7 architectures.
+
+#### Resources
+
+IoT devices are typically more resource constrained than cloud infrastructure hosts. The IoT Agent is built to have a minimal footprint and consume minimal network bandwidth.
+
+Exact resource requirements depend on usage. Datadog found the following when testing the IoT Agent (v7.20) internally:
+
+- CPU: 0.5% (on a VM with 2 Intel Xeon VCPUs)
+- Memory: 36 MB
+- Network bandwidth: 237 bps up / 79 bps down
+- Disk: 63 MB
+
+### Installation
+
+#### Automatic
+
+To automatically download and install the correct IoT Agent for your operating system and chipset architecture, use the following command:
+
+```shell
+DD_API_KEY=<YOUR_DD_API_KEY> DD_AGENT_MAJOR_VERSION=7 DD_AGENT_FLAVOR=iot-agent bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
```

Comment:
Noticed an error while testing this. Should be: DATADOG_AGENT_FLAVOR=datadog-iot-agent

Reply:
Should be `DD_AGENT_FLAVOR=datadog-iot-agent`, and we are changing the URL for that script also, let me fix it.